Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -5,7 +5,7 @@ import BraintreeCore
 #endif
 
 ///  Options for the PayPal Vault flow.
-@objcMembers public class BTPayPalVaultRequest: BTPayPalRequest {
+@objcMembers open class BTPayPalVaultRequest: BTPayPalRequest {
```

Comment:
This is now needs to be `open` because we are subclassing it outside of it's defining module